### PR TITLE
docker-compose: add docker-compose ps

### PR DESCRIPTION
**Pull request by ninearif**
add "docker-compose ps command" list all running containers.

- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by schneiderl** [comment]
I believe we should have this example at the beginning of the page as it is pretty commonly used.

**Comment by sbrl** [comment]
Oops, looks like that comment isn't attached to a line @schneiderl?

**Comment by schneiderl** [comment]
Thank you for pointing that out @sbrl

File: pages/common/docker-compose.md [hunk 1]
```diff
@@ -26,3 +26,7 @@
 - Follow logs for all containers:
 
 `docker-compose logs --follow`
+
+- Lists all running containers:
```
[the remark this answers is not in the document]
Comment (ninearif) [reply pages/common/docker-compose.md]: I agree with you.

**Comment by ninearif** [comment]
@schneiderl @sbrl 
I created a new PR, let's discuss here #3507

**Comment by sbrl** [comment]
Erm no need to open a new PR @ninearif - you can just push more commits to the branch that you've created the PR on to update it.

This said, if you've created a new PR we should close this one?

**Pull request closed without merging**